src/core.ts

```diff
@@ -122,15 +109,15 @@ export function createStarport<T extends Component>(
 
       onMounted(async() => {
         if (context.el) {
-          if (process.env.NODE_ENV === 'development')
+          if (isDevelopment())
```

Comment:
This is important to leave as a literal for bundler to do const folding and remove development code in production. This refactor disables that capability.